HRIS.Services.Tests/Services/AuthServiceUnitTest.cs

```diff
@@ -57,232 +55,8 @@ public AuthServiceUnitTest()
-
-        var employeeDto = new EmployeeDto
-        {
-            Id = 1,
-            EmployeeNumber = "001",
-            TaxNumber = "34434434",
-            EngagementDate = new DateTime(),
-            TerminationDate = new DateTime(),
-            PeopleChampion = null,
-            Disability = false,
-            DisabilityNotes = "None",
-            Level = 4,
-            EmployeeType = new EmployeeTypeDto { Id = 1, Name = "Developer" },
-            Notes = "Notes",
-            LeaveInterval = 1,
-            SalaryDays = 28,
-            PayRate = 128,
-            Salary = 100000,
-            Name = "Dotty",
-            Initials = "D",
-            Surname = "Missile",
-            DateOfBirth = new DateTime(),
-            CountryOfBirth = "South Africa",
-            Nationality = "South African",
-            IdNumber = "5522522655",
-            PassportNumber = " ",
-            PassportExpirationDate = new DateTime(),
-            PassportCountryIssue = "South Africa",
-            Race = Race.Black,
-            Gender = Gender.Female,
-            Email = "dm@retrorabbit.co.za",
-            PersonalEmail = "test@gmail.com",
-            CellphoneNo = "0123456789",
-            PhysicalAddress = employeeAddressDto,
-            PostalAddress = employeeAddressDto
-        };
-
-        var email = "dm@retrorabbit.co.za";
-
-        var employeeDtoList = new List<Employee>
-        {
-            new(employeeDto, employeeTypeDto1)
-        };
-
-        var roles = new Dictionary<string, List<string>>
-        {
-            {
-                "Admin", new List<string> { "Create", "Read", "Update", "Delete" }
-            }
-        };
-
-        _unitOfWork.Setup(r => r.Employee.Get(It.IsAny<Expression<Func<Employee, bool>>>()))
-                   .Returns(employeeDtoList.AsQueryable().BuildMock());
-        _employeeService.Setup(x => x.SaveEmployee(employeeDto)).ReturnsAsync(employeeDto);
-        _roleAccessLinkService.Setup(x => x.GetRoleByEmployee(email)).ReturnsAsync(roles);
-
-        _configuration.Setup(c => c["Auth:Key"]).Returns("34bDExFk0WHe$GaXn%0xcZY3^fT^#J05");
-        _configuration.Setup(c => c["Auth:Expires"]).Returns("30");
-        _configuration.Setup(c => c["Auth:Issuer"]).Returns("YourIssuer");
-        _configuration.Setup(c => c["Auth:Audience"]).Returns("YourAudience");
-
-        var token = await _authService.RegisterEmployee(employeeDto);
-
-        Assert.NotNull(token);
-    }
-
-    [Fact]
-    public async Task GetUserRoles()
-    {
-        var email = "dm@retrorabbit.co.za";
-        var expectedRoles = new Dictionary<string, List<string>>
-            { { "Admin", new List<string> { "Create", "Read", "Update", "Delete" } } };
-        _roleAccessLinkService.Setup(x => x.GetRoleByEmployee(email)).ReturnsAsync(expectedRoles);
-
-        var roles = await _authService.GetUserRoles(email);
-
-        Assert.Equal(expectedRoles, roles);
-    }
+    // TODO: update this unit test
```

Comment:
Why did you delete this unit test?

Reply:
The Auth Service has been completely changed. There is no proper unit test written for the revamped service yet.